Podfile

```diff
@@ -23,5 +25,18 @@ target 'ParselyTracker' do
     inherit! :search_paths
     # Pods for testing
   end
+end
+
+def make_pods_adopt_app_deployment_target(installer:, app_deployment_target:)
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |configuration|
+      delopyment_key = 'IPHONEOS_DEPLOYMENT_TARGET'
+      pod_deployment_target = Gem::Version.new(configuration.build_settings[delopyment_key])
+      configuration.build_settings.delete(delopyment_key) if pod_deployment_target <= app_deployment_target
+    end
+  end
+end
 
+post_install do |installer|
+  make_pods_adopt_app_deployment_target(installer: installer, app_deployment_target: APP_IOS_DEPLOYMENT_TARGET)
```

Comment:
This was necessary to make SwiftyJSON compile in Xcode 14.3 due to the change in deployment target. 

<img width="510" alt="image" src="https://user-images.githubusercontent.com/1218433/231917037-e8091838-da41-459b-8448-5e651b1f2022.png">